packages/wrangler/src/__tests__/deploy.test.ts

```diff
@@ -6922,7 +6922,8 @@ addEventListener('fetch', event => {});`
 			`);
```

Comment:
Delete this test if it is no longer relevant?
Did you just keep it here while the product change was discussed?